### PR TITLE
added README.md file

### PR DESCRIPTION
**Pull request by OlesiaKapustynska**

### CODE REVIEW

**Review by PashaTerletskyi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# Cinema app (in progress...)
+## Cinema app 
+This project is the implementation of the main functions of interaction between the cinema and the user.
+![pic](https://www.pngkit.com/png/full/10-100286_cinema-png-image-film.png)
+
+### Technologies used:
+- Spring - Core / MVC / Web / Security
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
- Spring - Core / MVC / Security
```

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+- Hibernate
+- Database - MySQL
+- Packaging - Apache Maven 
+- Tomcat
+***
+### Setup:
+1. Clone the project into your local directory and then open it in an IDE.
+1. Fill in the fields (db.url, db.user and db.password) in the /src/main/resources/db.properties. In this project using MySQL DB, if you use another database you need to change field "db.driver" and dependencies in web.xml.
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
1. Fill in the fields (db.url, db.user and db.password) in the `/src/main/resources/db.properties`. In this project we are using MySQL DB, if you use another database you need to change field "db.driver" and dependencies in `web.xml`.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+1. Configuration Local Tomcat: deployment - war_exploded, context address - "/"
```
Comment (KseniiaMakarova) [inline README.md]:
```suggestion
1. Configurate a Local Tomcat server: deployment - war_exploded, context address - "/"
```

**Review by KseniiaMakarova — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (OlesiaKapustynska) [reply README.md]: done ✔️

File: README.md (hunk 2 above)
Comment (OlesiaKapustynska) [reply README.md]: done ✔️

File: README.md (hunk 3 above)
Comment (OlesiaKapustynska) [reply README.md]: done ✔️

**Pull request merged**